python/ray/tune/tune_config.py

```diff
@@ -59,3 +66,4 @@ class TuneConfig:
     max_concurrent_trials: Optional[int] = None
     time_budget_s: Optional[Union[int, float, datetime.timedelta]] = None
     reuse_actors: Optional[bool] = None
+    chdir_to_trial_dir: bool = True
```

Comment:
When I do a follow-up PR to allow Trainers to also not change the working dir, this config would need to be moved to `air.RunConfig` (something shared between Trainers and Tuners).